src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+
+import Icon from '~core/Icon';
+import Heading from '~core/Heading';
+import HookedUserAvatar from '~users/HookedUserAvatar';
+import { Colony } from '~data/index';
+import { Address } from '~types/index';
+import { getMainClasses } from '~utils/css';
+
+import useAvatarDisplayCounter from '../../../hooks/useAvatarDisplayCounter';
+
+import styles from './VoteResultsWidget.css';
+
+interface Appearance {
+  theme?: 'approve' | 'disapprove';
+}
+
+interface Props {
+  colony: Colony;
+  appearance?: Appearance;
+  value?: number;
+  title: string;
```

Comment:
This should be a message descriptor! 

We're not really using plain string values anywhere in dapp